Makefile

```diff
@@ -330,6 +331,7 @@ endif
 $(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi: msidir
 	candle.exe -arch x64 -ext WixUtilExtension -o $(PACKAGE_DIR)/msi/ $(PACKAGE_DIR)/msi/*.wxs
 	cd $(PACKAGE_DIR)/msi && light.exe -ext WixUIExtension -ext WixUtilExtension -sacl -spdb -sice:ICE61 -sice:ICE69 -out ../../../$@ *.wixobj
+	pushd $(BUILD_DIR)/windows-amd64 && sha256sum crc-windows-amd64.msi > crc-windows-amd64.msi.sha256sum && popd
```

Comment:
Fwiw, looks like this can be:
```
pushd $(@D) && sha256sum $(@F)>$(@F).sha256sum && popd
```
see https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html#Automatic-Variables
You don't really need `pushd`/`popd`, this can be just `cd $(@D); sha256sum ...`. I think this runs in a subshell, so the directory is unchanged for the following commands.

This could even go to its own Makefile rule:
```
%.sha256sum: %
    cd $(@D); sha256sum $(^F)>$(@F)

msi: $(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi $(BUILD_DIR)/windows-amd64/crc-windows-amd64.msi.sha256sum
```